sdk/js/src/util/bytes.js

```diff
@@ -0,0 +1,46 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Converts hex encoded string to base64.
+ *
+ * @param {string} str - Hex encoded string.
+ * @returns {string} - `str` base64 encoded.
+ */
```

Comment:
This file is a duplicate of `console/lib/bytes` but it doesn't seem wise to import the console functions here.

Reply:
Maybe it would be wise to make the console use this SDK function instead of the console one in a tech debt PR in the future.